python_src/src/lamp_py/runtime_utils/alembic_migration.py

```diff
@@ -17,23 +15,12 @@ def run_alembic_migration(db_name: str) -> None:
     logging.info("running alembic with config file %s", alembic_cfg_file)
 
     if db_name == "performance_manager":
-        # if database is clean, call to DatabaseManager will trigger
-        # SqlBase.metadata.create_all(self.engine)
-        # creating all database tables
-        db_manager = DatabaseManager()
+        pass
     else:
         raise NotImplementedError(f"Migration for {db_name} not implemented.")
 
     # load alembic configuation for db_name
     alembic_cfg = Config(alembic_cfg_file, ini_section=db_name)
 
-    # check if alembic_version table exists in rds
-    try:
-        db_manager.select_as_list("SELECT 1 FROM alembic_version")
-    except Exception as _:
-        # if no version table, create version table and stamp with current head
-        # this assumes the head matches current rds state
-        command.stamp(alembic_cfg, revision="head")
-
```

Comment:
This was only needed for alembic's initial run against our DEV RDS to create `alembic_version` table. 

This is no longer needed.